Postman/production.postman_environment.json

```diff
@@ -0,0 +1,52 @@
+{
+  "id": "6a870495-179d-4b9c-9117-d6442ebcc6b1",
```

Comment:
there are more production environments - I think this one is EU

Reply:
ok, I will add other production environments when I know the links